include/fes/wave.hpp

```diff
@@ -1049,13 +1049,14 @@ class SN4 : public Wave {
 /// <tr><th>V</th><th>u</th><th>Factor-f</th></tr>
 /// <tr><td>@f$4T - 2s + 2h@f$</td>
 /// <td>@f$+2\xi - 2\nu@f$</td>
-/// <td>@f$f(M_2)@f$</td></tr>
+/// <td>@f$f(M_2)^2@f$</td></tr>
 /// </table>
+/// @note Shureman: %Table 2a, Page 167
 class MS4 : public Wave {
  public:
   constexpr MS4()
       : Wave(kMS4, kShortPeriod, false, 4, -2, 2, 0, 0, 0, 0, 2, -2, 0, 0,
-             &angle::Astronomic::f_m2) {}
```

Comment:
In the Schureman, there is an error in the nodal factor associated with this wave. It is f(M2) that should be taken into account. Can you restore the previous version on the code here? You can add a text in the documentation if you want to mention it. You can refer to this reference if you like: Pugh, D.T. (1987). [Tides, surges and mean sea-level - a handbook for engineers and scientists (Page 111, Table 4.4)](https://refmar.shom.fr/sites/default/files/2024-07/Pugh%2C%20D.T.%20%281987%29.%20Tides%2C%20surges%20and%20mean%20sea-level%20-%20a%20handbook%20for%20engineers%20and%20scientists.%20Wiley%2C%20Chichester.pdf).

Reply:
Fantastic. Thanks for clarifying the source of the correct value. I have amended my code and included the Pugh citation.